libs/components/lookup/testing/src/lookup/lookup-harness.spec.ts

```diff
@@ -6,12 +6,26 @@ import { LookupHarnessTestComponent } from './fixtures/lookup-harness-test.compo
 import { LookupHarnessTestModule } from './fixtures/lookup-harness-test.module';
 import { SkyLookupHarness } from './lookup-harness';
 
-async function setupTest(options: { dataSkyId: string }) {
+async function setupTest(options: {
+  dataSkyId: string;
+  selectionDescriptor?: string;
+  enableCustomTemplate?: boolean;
+}) {
   await TestBed.configureTestingModule({
     imports: [LookupHarnessTestModule],
   }).compileComponents();
 
   const fixture = TestBed.createComponent(LookupHarnessTestComponent);
+  fixture.componentInstance.showMoreConfig.nativePickerConfig = Object.assign(
+    { selectionDescriptor: options.selectionDescriptor },
+    fixture.componentInstance.showMoreConfig.nativePickerConfig
+  );
+  if (options.enableCustomTemplate) {
+    fixture.componentInstance.showMoreConfig.nativePickerConfig = Object.assign(
+      { itemTemplate: fixture.componentInstance.showMoreSearchResultTemplate },
+      fixture.componentInstance.showMoreConfig.nativePickerConfig
+    );
+  }
```

Comment:
Mind putting new lines before and after this `if` block?

Reply:
Done